CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Version changelog
 
+## 1.24.1
+ * Fixed verification of workspace reachability by using scim/me which is always available  ([#2618](https://github.com/databricks/terraform-provider-databricks/pull/2618)).
```

Comment:
Missing newline between header and body.

Reply:
Added, thanks for the review.